src/librustc_resolve/diagnostics.rs

```diff
@@ -354,6 +354,13 @@ impl<'a> Resolver<'a> {
                     span, "defaulted type parameters cannot be forward declared".to_string());
                 err
             }
+            ResolutionError::SelfInTyParamDefault => {
+                let mut err = struct_span_err!(self.session, span, E0734,
```

Comment:
Formatting should be as in the arm below.

Reply:
The formatting does match that of the arm *above*, which is the closest analogy in terms of the intent here (since the use of a `Self` param is semantically like a use of a forward-declaration.)

Nonetheless, I will reindent the new arm.